### DISCUSSION TITLE
[Prettier Plugin] License of prettier-plugin-tailwindcss

### DISCUSSION THREAD

**Discussion by hyunbinseo** [tailwindlabs/tailwindcss#7784] [Help] [answered]
What is the license of [prettier-plugin-tailwindcss](https://github.com/tailwindlabs/prettier-plugin-tailwindcss)?

It is not specified in the repository.

**Comment by adamwathan** [comment, marked as answer]
Added! It's MIT 👍

**Reply by hyunbinseo** [reply]
In [npm](https://www.npmjs.com/package/prettier-plugin-tailwindcss), the license value is still `none`. Could this be fixed?

Having hard time automating OSS Notice generation.
